pkg/webhook/controlplane/ensurer.go

```diff
@@ -23,10 +23,8 @@ import (
 
 	"github.com/Masterminds/semver"
 	"github.com/coreos/go-systemd/v22/unit"
-	"github.com/gardener/gardener/extensions/pkg/controller/csimigration"
```

Comment:
https://github.com/gardener/gardener-extension-provider-aws/blob/c9a30abe43a0523a9b25ab85935e141cc7ba6cf4/pkg/webhook/controlplane/ensurer.go#L153-L161 could be also cleaned up.